### PR TITLE
Update scaffold readme

### PR DESCRIPTION
**Pull request by nicholemaureen**
Update tag instructions for scaffold to include versioning syntax and use instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,13 @@ Likewise, if there is a new great idea that we should incorporate into this
 scaffold, any Team Member can file an issue and bring the idea forward for
 review and inclusion.
 
-### Using and Creating Tags
+### Creating Tags
+
 New changes that get merged to master should also receive a new tagged release.
 
+Follow the conventions outlined here: http://semver.org/ to determine the new 
+release number you should bump up to,
```
Comment (raubin) [inline README.md]: replace comma with period

File: README.md [hunk 2]
```diff
@@ -45,12 +49,21 @@ way you might write a quick commit message. Your release notes should contain a
...
 
+### Using a Tagged Release
+
+When adding scaffold to a new project, make sure you use the latest tagged release 
+instead of just cloning master (as we did previously).
+
+To grab the lastest tagged release, navigate to the 'Releases' tab for this repo and download
+the most recent version.
+
+
```
Comment (raubin) [inline README.md]: looks like extra line break

File: README.md [hunk 3]
```diff
@@ -45,12 +49,21 @@ way you might write a quick commit message. Your release notes should contain a
...
 For more information on tags, check https://git-scm.com/book/en/v2/Git-Basics-Tagging.
 
 ## Getting Started
 
+### Using a Tagged Release
+
+When adding scaffold to a new project, make sure you use the latest tagged release 
+instead of just cloning master (as we did previously).
+
+To grab the lastest tagged release, navigate to the 'Releases' tab for this repo and download
```
Comment (marcaddeo) [inline README.md]: Technically 'Releases' isn't a tab :o

Comment (raubin) [reply README.md]: First one's a link, yah, but what's this thing called?
image

Comment (nicholemaureen) [reply README.md]: I think this one is okay to leave as 'tab', even though it technically isn't one. It indicates the general area to look for the link.

**Comment by nicholemaureen** [comment]
@raubin Updated!

File: README.md (hunk 3 above)
Comment (balbuf) [reply README.md]: "section" ? 🤷‍♂️

**Comment by raubin** [comment]
🚀 to the 🌑! :shipit: /approved

**Pull request merged**